ArchiverAccess/log_file_generator.py

```diff
@@ -53,18 +46,18 @@ def create_log(headers, columns, time_period, filename_template="default.log", h
         config_builder.table_column(column_header, column_template)
 
     adfc = ArchiveDataFileCreator(config_builder.build(), archiver_data_source)
-    adfc.write(time_period)
+    adfc.write_complete_file(time_period)
 
 if __name__ == '__main__':
```

Comment:
It took 121 seconds to run this file. I am not sure what we consider reasonable so I will just leave this here